toolchest_client/api/query.py

```diff
@@ -339,7 +341,7 @@ def _download(self, output_path, output_type):
 
         try:
             self.output_s3_uri, output_file_keys = get_download_details(self.PIPELINE_SEGMENT_INSTANCE_ID)
-            if output_path:
+            if output_path and not path_is_s3_uri(output_path):
```

Comment:
Adding this condition seems like it should skip the need to edit `unpack_files()` in `unpack.py`; see the other comment.

Reply:
This is mainly to prevent the need to update `download()` and may change in the future per my other comment.